package.json

```diff
@@ -28,16 +28,15 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@hapi/boom": "7.x.x",
-    "db-errors": "0.2.x"
+    "@hapi/boom": "9.x.x"
   },
   "devDependencies": {
     "objection": "2.x.x",
     "coveralls": "3.x.x",
-    "@hapi/code": "6.x.x",
-    "@hapi/lab": "20.x.x"
+    "@hapi/code": "8.x.x",
+    "@hapi/lab": "22.x.x"
   },
   "peerDependencies": {
-    "objection": ">=1 <3"
+    "objection": ">=2 <3"
```

Comment:
This would be considered a breaking (major) change upon release, which seems okay 👍